language/guidelines/user-experience/performance/index.md

```diff
@@ -0,0 +1,73 @@
+---
+type: guideline
+title: Performance
+resources:
+  links:
+    - name: "North: Performance"
+      source: http://pointnorth.io/#performance
+    - name: "Have Your Cake and Eat it Too: A Primer on Performance"
+      source: http://una.im/slides-nom-nom-perf/#/
+    - name: "Our need for web speed: It's about neuroscience, not entitlement"
+      source: http://www.webperformancetoday.com/2012/03/21/neuroscience-page-speed-web-performance/
+    - name: "Powers of 10: Time Scales in User Experience"
+      source: http://www.nngroup.com/articles/powers-of-10-time-scales-in-ux/
+    - name: "Jank Free"
+      source: http://jankfree.org/
+    - name: "Web Page Test"
+      source: http://www.webpagetest.org/
+---
+
+Performance is an integral part of a user's experience no matter the medium that interface takes. Web delivered interfaces, native interfaces, application programming interfaces; whenever a user interacts with a product, performance always needs to be a top concern. Slow performing interfaces have been shown to [cost money and conversions](http://www.fastcompany.com/1825005/how-one-second-could-cost-amazon-16-billion-sales) and [affect a company's bottom line](https://blog.kissmetrics.com/loading-time/). More than that, performance is about respecting our users and the way humans perceive and react to the world around them.
+
+## The Neuroscience of Performance
+
+In Jakob Nielsen's book _Usability Engineering_, he [describes three important limits in human response time](http://www.nngroup.com/articles/response-times-3-important-limits/):
+
+> * **0.1 second** is about the limit for having the user feel that the system is **reacting instantaneously**, meaning that no special feedback is necessary except to display the result.
+>* **1.0 second** is about the limit for the **user's flow of thought** to stay uninterrupted, even though the user will notice the delay. Normally, no special feedback is necessary during delays of more than 0.1 but less than 1.0 second, but the user does lose the feeling of operating directly on the data.
+> * **10 seconds** is about the limit for **keeping the user's attention** focused on the dialogue. For longer delays, users will want to perform other tasks while waiting for the computer to finish, so they should be given feedback indicating when the computer expects to be done. Feedback during the delay is especially important if the response time is likely to be highly variable, since users will then not know what to expect.
```

Comment:
is the approximate limit

because otherwise users will not know what to expect.